code/datums/mood_events/generic_negative_events.dm

```diff
@@ -154,6 +154,10 @@
 	description = "I'm nervous and on edge and I can't stand still!!"
 	mood_change = -2
 
+/datum/mood_event/choke
+	description = "I CAN'T BREATH!!!"
```

Comment:
```suggestion
	description = "I CAN'T BREATHE!!!"
```

although "i can't breath" is quite funny so feel free to ignore this

Reply:
fcuk

Reply:
hii